packages/flutter/lib/src/widgets/sliver_fill.dart

```diff
@@ -122,6 +122,8 @@ class _RenderSliverFractionalPadding extends RenderSliverEdgeInsetsPadding {
       assert(viewportFraction >= 0),
       _viewportFraction = viewportFraction;
 
+  SliverConstraints _lastResolvedConstraints;
```

Comment:
Instead of holding on to the old constraints, could we just overwrite the "constraints" getter and invalidate the old resolution there when new constraints are supplied?

Reply:
I think the `constraints` setter is not exposed? I'm not sure how I can detect the change with the getter along